source/common/router/router.cc

```diff
@@ -193,6 +193,33 @@ FilterUtility::finalTimeout(const RouteEntry& route, Http::HeaderMap& request_he
   return timeout;
 }
 
+FilterUtility::HedgingParams FilterUtility::finalHedgingParams(const RouteEntry& route,
+                                                               Http::HeaderMap& request_headers,
+                                                               uint64_t random_value) {
+  HedgingParams hedgingParams;
+  hedgingParams.initial_requests_ = route.hedgePolicy().initialRequests();
```

Comment:
This is not implemented yet, right? (It's marked as such in the docs). If so, can we remove this code for now from the struct and here?